rework_tests/unit/estimators/iterative/pruners/test_prior_pruner.py

```diff
@@ -159,7 +159,7 @@ def test_prune_does_not_remove_last_component(dummy_components):
     state = PipelineState(X=np.array([]), H=None, prev_mixture=None, curr_mixture=initial_mixture, error=None)
 
     # After the first prune, one component will remain, which should not be removed
-    new_state = pruner.prune(state)
+    new_state, removed_components_indices = pruner.prune(state)
```

Comment:
Name variables that dont used as `_`

```suggestion
    new_state, _ = pruner.prune(state)
```